sale_coupon_advanced/models/sale_coupon.py

```diff
@@ -14,6 +14,21 @@ class SaleCouponProgram(models.Model):
         help="Apply only on the first order of each client matching the conditions",
     )
 
+    first_n_orders_only = fields.Integer(
+        help="Maximum number of sales orders of the customer in which reward \
+         can be provided",
+        string="Apply only on the next ",
```

Comment:
```suggestion
        string="Amount of orders to be applied",
```
there is already string with similar name